scripts/run.sh

```diff
@@ -28,24 +28,29 @@ OS="lin"
 ENV_DIR="$OUTPUT_DIR/env"
 DOCKER_DIR="$OUTPUT_DIR/docker"
 
-# Initialize UID/GID which will be used to run services
+# Initialize UID/GID which will be used to run services identically over all the OS...
 if ! grep -q "^LOCAL_UID=" $ENV_DIR/uid.env 2>/dev/null || ! grep -q "^LOCAL_GID=" $ENV_DIR/uid.env 2>/dev/null
 then
     LUID="LOCAL_UID=`id -u $USER`"
+    [ "$LUID" == "LOCAL_UID=0" ] && LUID="LOCAL_UID=65534"
     LGID="LOCAL_GID=`id -g $USER`"
+    [ "$LGID" == "LOCAL_GID=0" ] && LGID="LOCAL_GID=65534"
```

Comment:
I switch from `0` to `65534` directly in `uid.env` so that it's more clear for the user what UID/GID are used by default.